Remotion/Web/Core/UI/Controls/WebTabCollection.cs

```diff
@@ -123,7 +124,8 @@ protected override void OnClear ()
       base.OnClear();
       for (int i = 0; i < InnerList.Count; i++)
       {
-        WebTab tab = (WebTab) InnerList[i];
+        // TODO: Add not null assertion
```

Comment:
If we guard all usages of InnerList where an item is added, we can't have anything but not-null in there,.